src/lib/common/include/sol-platform.h

```diff
@@ -73,9 +73,10 @@ extern "C" {
 const char *sol_platform_get_board_name(void);
 
 /**
- * @brief Retrieves, in @a id, the machine-id present in the file system.
+ * @brief Retrieves the machine-id present in the file system as a string.
  *
```

Comment:
It is not clear what is machine-id. OIC defines platform ID and device ID. Only device ID is UUID. Platform ID is opaque (can be UUID, VIN, etc).